### PR TITLE
expand disk using console and cli

### LINKED ISSUE #375: Add documentation for the disk expand the feature on ESC
See: https://github.com/EventStore/documentation/pull/367/files#r649922994

### PR DESCRIPTION
**Pull request by ylorph**

### CODE REVIEW

File: docs/cloud/ops/disk_expand.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+﻿# Expanding disks
+
+Disk can be extended, to accomodate database growth,through the [Cloud Console](https://console.eventstore.cloud/) and the [Event Store Cloud CLI](https://github.com/EventStore/esc)
```
Comment (oskardudycz) [inline docs/cloud/ops/disk_expand.md]:
```suggestion
A disk can be expanded on-demand, to accommodate database growth, through the [Cloud Console](https://console.eventstore.cloud/) and the [Event Store Cloud CLI](https://github.com/EventStore/esc)
```

File: docs/cloud/ops/disk_expand.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+See also the cloud [sizing guide](/cloud/provision/cloud-instance-guidance/#sizes) for general guidance. 
+
+
+::: note 
+Limitations:  
+* on AWS expanding disks is subject to a rate limit, see [here](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_ModifyVolume.html) for more information.  
+* on Azure, it is currently not possible to expand disks. To increase disk size you need to perform a backup and restore or replicate to a cluster with larger disks.
+:::
```
Comment (oskardudycz) [inline docs/cloud/ops/disk_expand.md:11]: **QUESTION:** No limitations on GCP?

File: docs/cloud/ops/disk_expand.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+﻿# Expanding disks
+
+Disk can be extended, to accomodate database growth,through the [Cloud Console](https://console.eventstore.cloud/) and the [Event Store Cloud CLI](https://github.com/EventStore/esc)
+
+See also the cloud [sizing guide](/cloud/provision/cloud-instance-guidance/#sizes) for general guidance. 
+
+
```
Comment (oskardudycz) [inline docs/cloud/ops/disk_expand.md]: **SUGGESTION:** There are redundant double empty lines around the note.

File: docs/cloud/ops/disk_expand.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+Limitations:  
+* on AWS expanding disks is subject to a rate limit, see [here](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_ModifyVolume.html) for more information.  
+* on Azure, it is currently not possible to expand disks. To increase disk size you need to perform a backup and restore or replicate to a cluster with larger disks.
+:::
+
+
+## Using the Cloud Console
+
+To expand disks in the console, navigate to the clusters view and click on the _Expand Disks_ icon.
+::: card
```
Comment (oskardudycz) [inline docs/cloud/ops/disk_expand.md:17]: **SUGGESTION:** Might be worth adding empty lines around cards.

File: docs/cloud/ops/disk_expand.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+::: note 
+Limitations:  
+* on AWS expanding disks is subject to a rate limit, see [here](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_ModifyVolume.html) for more information.  
+* on Azure, it is currently not possible to expand disks. To increase disk size you need to perform a backup and restore or replicate to a cluster with larger disks.
+:::
+
+
+## Using the Cloud Console
+
+To expand disks in the console, navigate to the clusters view and click on the _Expand Disks_ icon.
```
Comment (oskardudycz) [inline docs/cloud/ops/disk_expand.md:15]: **QUESTION:** Shall we be using capital letters for console or only if we're giving the full name? The same question applies to the command line below.

File: docs/cloud/ops/disk_expand.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+On the detail page specify the new disk size and click on _Expand cluster disk_.
+::: card
+![cluster_expand_detail](./images/disk_expand_detail.png)
+:::
+
+## Using the command line
+
+To expand disks with  the command line, use the `cluster expand` command, where `--id` is the cluster id. 
+
+``` bash
```
Comment (oskardudycz) [inline docs/cloud/ops/disk_expand.md]:
```suggestion
```bash
```

**Comment by oskardudycz** [comment]
It might be worth adding a link to these docs: https://developers.eventstore.com/cloud/faq/cluster-provisioning.html#are-there-plans-to-support-automatic-disk-resize.

File: docs/cloud/ops/disk_expand.md (hunk 2 above)
Comment (ylorph) [reply docs/cloud/ops/disk_expand.md:11]: nope

**Review by oskardudycz — APPROVED** [review]

**Comment by oskardudycz** [comment]
Looks good 👍

**Review by alexeyzimarev — APPROVED** [review]

**Pull request merged**